composer/datasets/in_context_learning_evaluation.py

```diff
@@ -1343,105 +1576,159 @@ def get_icl_task_dataloader(
         max_seq_len: int,
         pad_tok_id: int,
         num_fewshot: int,
-        prompt_string: str,  # e.g. 'translate english to french:'
-        example_delimiter: str,  # e.g. '\n'
-        continuation_delimiter: str = '',
+        prompt_string: str = '',  # e.g. 'translate english to french:'
+        example_delimiter: str = '\n',  # e.g. '\n'
+        continuation_delimiter: str = ' ',
         destination_path: str = '',
         question_prelimiter: str = '',  # e.g. 'Question: '
         fewshot_random_seed: int = 1234,
         pass_at_k: int = 1,
-        generations_per_sample: int = 1,
+        generations_per_sample: int = 20,
```

Comment:
This is a change to default, what is the justification?

Reply:
Same as above (the change is meant to mirror the default that [is set in llm-foundry](https://github.com/mosaicml/llm-foundry/blob/main/llmfoundry/utils/builders.py#L486).)